src/tera.rs

```diff
@@ -19,6 +19,9 @@ pub struct Tera {
     pub templates: HashMap<String, Template>,
     pub filters: HashMap<String, FilterFn>,
     pub testers: HashMap<String, TesterFn>,
+    // Which extensions does Tera automatically autoescape on.
+    // Defaults to ["html", "htm", "xml"]
+    pub autoescape_extensions: Vec<String>,
```

Comment:
Why is this a `Vec<String>` and not just a `Vec<&'static str>`? In fact, does it need to be a `Vec` at all? Why not a `[&'static str; 3]`? You'd need a `Vec` if you want to allow users to add their own extensions.